requirements.csv

```diff
@@ -21,5 +21,9 @@ python_not_in_db,packaging
 python_not_in_db,importlib_resources
 vsajip_not_in_db,python-gnupg
 anthonyharrison_not_in_db,lib4sbom
+<<<<<<< HEAD
+the_purl_authors_not_in_db,packageurl-python
+=======
```

Comment:
```suggestion
```

Looks like your conflict resolution went wrong.  You'll need to remove these lines and the one below to resolve it correctly.  

If git conflict resolution isn't something you've done before, you might find this article helpful: https://www.atlassian.com/git/tutorials/using-branches/merge-conflicts